components/developer/rust/manifests/sample-manifest.p5m

```diff
@@ -0,0 +1,2501 @@
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/clzdi2.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/clzsi2.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/clzti2.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/cmpdi2.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/cmpti2.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/comparedf2.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/comparesf2.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/comparetf2.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/cpu_model.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/ctzdi2.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/ctzsi2.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/ctzti2.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/divdc3.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/divdf3.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/divdi3.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/divmoddi4.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/divmodsi4.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/divsc3.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/divsf3.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/divsi3.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/divtc3.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/divtf3.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/divti3.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/divxc3.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/emutls.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/enable_execute_stack.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/lib/builtins/eprintf.c
+file path=usr/lib/$(MACH64)/rustlib/src/rust/src/libcompiler_builtins/compiler-rt/
```

Comment:
Should it go to /usr/share/zsh/vendor-functions/ ?

Reply:
I'll move it.